mmdeploy/codebase/mmrotate/models/roi_heads/roi_trans_roi_head.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+from mmdeploy.core import FUNCTION_REWRITER
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    'mmrotate.models.roi_heads.roi_trans_roi_head'
+    '.RoITransRoIHead.simple_test')
+def roi_trans_roi_head__simple_test(ctx, self, x, proposals, img_metas,
```

Comment:
we should keep the argument `proposals` same as the original `proposal_list` in https://github.com/open-mmlab/mmrotate/blob/c62f148fcc2c8253218e67f2277cb8770bcd7df0/mmrotate/models/roi_heads/roi_trans_roi_head.py#L266

Reply:
I'll try it.